base/bucket.go

```diff
@@ -106,18 +106,19 @@ type CouchbaseBucketType int
 
 // Full specification of how to connect to a bucket
 type BucketSpec struct {
-	Server, BucketName, FeedType  string
-	Auth                          AuthHandler
-	CouchbaseDriver               CouchbaseDriver
-	Certpath, Keypath, CACertPath string         // X.509 auth parameters
-	TLSSkipVerify                 bool           // Use insecureSkipVerify when secure scheme (couchbases) is used and cacertpath is undefined
-	KvTLSPort                     int            // Port to use for memcached over TLS.  Required for cbdatasource auth when using TLS
-	MaxNumRetries                 int            // max number of retries before giving up
-	InitialRetrySleepTimeMS       int            // the initial time to sleep in between retry attempts (in millisecond), which will double each retry
-	UseXattrs                     bool           // Whether to use xattrs to store _sync metadata.  Used during view initialization
-	ViewQueryTimeoutSecs          *uint32        // the view query timeout in seconds (default: 75 seconds)
-	BucketOpTimeout               *time.Duration // How long bucket ops should block returning "operation timed out". If nil, uses GoCB default.  GoCB buckets only.
-	KvPoolSize                    int            // gocb kv_pool_size - number of pipelines per node. Initialized on GetGoCBConnString
+	Server, BucketName, FeedType     string
+	Auth                             AuthHandler
+	CouchbaseDriver                  CouchbaseDriver
+	Certpath, Keypath, CACertPath    string         // X.509 auth parameters
+	TLSSkipVerify                    bool           // Use insecureSkipVerify when secure scheme (couchbases) is used and cacertpath is undefined
+	KvTLSPort                        int            // Port to use for memcached over TLS.  Required for cbdatasource auth when using TLS
+	MaxNumRetries                    int            // max number of retries before giving up
+	InitialRetrySleepTimeMS          int            // the initial time to sleep in between retry attempts (in millisecond), which will double each retry
+	UseXattrs                        bool           // Whether to use xattrs to store _sync metadata.  Used during view initialization
+	ViewQueryTimeoutSecs             *uint32        // the view query timeout in seconds (default: 75 seconds)
+	ViewQueryMaxConcurrentOpsPerNode int            // maximum number of concurrent view / query operations (default: 100)
```

Comment:
I think `default: DefaultMaxConcurrentQueryOps` is probably better.  On a related note, I think 100 is probably too low of a default and should be updated from the previous default.  Given that we default MaxFileDescriptors to 5000, I think it's probably reasonable to set this default to 1000.